x/feeshare/ante/expected_keepers.go

```diff
@@ -0,0 +1,18 @@
+package ante
+
+// Used for the ante handler so we can properly send 50% of fees to dAPP developers via x/feeshare module
+
+import (
+	revtypes "github.com/classic-terra/core/x/feeshare/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type BankKeeper interface {
+	SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
+	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
+}
```

Comment:
Add this to your BankKeeper interface to fix the antehandler type issue:

SendCoinsFromModuleToModule(ctx sdk.Context, senderModule string, recipientModule string, amt sdk.Coins) error

This will also make the tests work

Reply:
I had modified the custom/auth/expected_keepers.go accidently it seems here. But still believe you are getting a local cache result, because `TestAnteTestSuite/TestIntegrationTaxExemption suite.go:77: test panicked: UnmarshalJSON cannot decode empty bytes` fails for me. But does not fail when I comment out the feeshare decorator.

I've got to look into it more, but have a busy day today. I suspect it's something small or a way the module needs to change to satisfy v44 in some aspect

Reply:
I spent most of the night testing and it seems to be related to your custom params usage. As such if you disable the GetParams call in your decorator the tests work. However overriding the stringer method for Params like the other modules does via "option (gogoproto.goproto_stringer) = false;" doesnt seem to change much. Maybe we can jump on a call and debug it together if you have time. you can reach me on discord @ Zaradar#1078

Reply:
Also another thing I noticed in the integration test is that you did not account for the extra BankKeeper your adding in @ https://github.com/Reecepbcups/lunc-core/blob/c7eb092c55947d8d30289968790e56f3bc946029/custom/auth/ante/integration_test.go#L143 , however I do believe you can avoid that argument all together if you just align the expected_keeper in custom/auth/ante and x/feeshare/ante :)